corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,14 @@
+/* globals hqDefine moment alert_user */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId) {
```

Comment:
Yeah. I originally meant in addition to. your reasoning against it makes sense. I forgot that most people have lots of forms open at once. Although could be nice to be able to hit save, get a notification and realize you have duplicate tabs open.